notebooks/MIRI/JWPipeNB-MIRI-MRS.ipynb

```diff
@@ -23,8 +23,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        nselfcal = len(selfcalfiles)
</pre><p>selfcalfiles is a required param and you're expecting a list, yes?  Why not something like </p><p> </p><p>for name in selfcalfiles:</p><p>	 with fits.open(name) as hdu:</p><p>          hdu.verify()</p><p>          if (hdu[0].header['CHANNEL'] == this_channel):</p><p>              asn['products'][0]['members'].append({'expname': name, 'exptype': 'selfcal'})</p><p>if selfcalfiles:</p><p>   write the asn</p><p> </p><p>same comment for the bgfiles lines above this</p><p>           </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwst-pipeline-notebooks/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
this formatted poorly, but you get the idea :)

Reply:
Looks like that should be functionally equivalent, just cuts down on some unnecessary variables and vertical space.  Done.

Reply:
yup, it's more pythonic too